<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddProd</class>
 <widget class="QDialog" name="AddProd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Product</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 248, 156);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="setPicBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set picture</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceLd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shopCb">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>190</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Articul</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="articulLd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>300</y>
      <width>191</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="approvBut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Approv</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
